components/opponent/wikis/rocketleague/player_ext_custom.lua

```diff
@@ -24,22 +30,65 @@ function PlayerExtCustom.fetchTeamHistoryEntry(resolvedPageName, date)
 		return
 	end
 	local conditions = {
-		'[[type::Notable]]',
+		'[[opponenttype::solo]]', -- can not use Opponent.solo due to circular requires
 		'[[pagename::' .. mw.title.getCurrentTitle().text:gsub(' ', '_') .. ']]',
-		'[[name::' .. resolvedPageName .. ']]',
+		'[[opponentname::' .. resolvedPageName .. ']]',
 	}
-	local datapoint = mw.ext.LiquipediaDB.lpdb('datapoint', {
+	local placement = mw.ext.LiquipediaDB.lpdb('placement', {
 		limit = 1,
 		conditions = table.concat(conditions, ' AND '),
-		query = 'information',
-	})[1]
-	if datapoint and Logic.isNotEmpty(datapoint.information) then
+		query = 'opponentplayers',
+	})[1] or {}
+
+	local team = (placement.opponentplayers or {}).p1team
+	if Logic.isNotEmpty(team) then
 		return {
 			joinDate = date,
 			leaveDate = date,
-			template = datapoint.information:lower(),
+			template = team:gsub('_', ' '):lower(),
 		}
 	end
 end
 
+--For specific uses only.
+---@param resolvedPageName string
+---@param date string|number|osdate?
+---@return string?
+function PlayerExtCustom.fetchTeamTemplate(resolvedPageName, date)
+	local entry = PlayerExtCustom.fetchTeamHistoryEntry(resolvedPageName, date)
+	return entry and TeamTemplate.resolve(entry.template, date) or nil
+end
+
+---@param pageName string
+---@param template string?
+---@param options {date: string|number|osdate?, useTimeless: boolean, fetchPlayer: boolean, savePageVar: boolean}
+---@return string?
+function PlayerExtCustom.syncTeam(pageName, template, options)
+	options = options or {}
+
+	template = PlayerExt.syncTeam(pageName, template, options)
+	if Logic.isNotEmpty(template) or options.fetchPlayer == false then return template end
+
+	local entry = PlayerExtCustom.fetchTeamHistoryEntry(pageName, options.date) --[[@as table]]
```

Comment:
If we are overriding this function as well and not only `fetchTeamHistoryEntry`, i'd suggest to rename that one, as it's not fetching history

Reply:
theoretically `fetchTeamHistoryEntry` could be called form an outside module
will have to double check

Reply:
no usage, so good idea